safeeyes/tests/test_core.py

```diff
@@ -0,0 +1,584 @@
+from collections import deque
+import datetime
+import logging
+import pytest
+
+from safeeyes import core
+from safeeyes import model
+
+import threading
+from time import sleep
+
+from unittest import mock
+
+class TestSafeEyesCore:
+    @pytest.fixture(autouse=True)
+    def set_time(self, time_machine):
+        time_machine.move_to(datetime.datetime.fromisoformat("2024-08-25T13:00:00+00:00"), tick=False)
+
+    @pytest.fixture(autouse=True)
```

Comment:
It is advised to store fixtures in `tests/conftest.py` file, especially such huge fixtures